projects/layout/components/cell/cell.directive.ts

```diff
@@ -31,12 +34,18 @@ class TuiCellStyles {}
     ],
     host: {
         tuiCell: '',
-        '[attr.data-size]': 'size || "l"',
+        '[attr.data-size]': 'size || options.size',
+        '[attr.data-height]': 'height || options.height',
     },
 })
 export class TuiCell {
+    protected readonly options = inject(TUI_CELL_OPTIONS);
+
     protected readonly nothing = tuiWithStyles(TuiCellStyles);
 
     @Input('tuiCell')
-    public size: TuiSizeL | TuiSizeS | '' = '';
+    public size: TuiCellOptions['size'] | '' = this.options.size;
+
+    @Input('tuiCellHeight')
+    public height: TuiCellOptions['height'] | '' = this.options.height;
```

Comment:
No reason for empty line here, there's no need to write `tuiCellHeight` empty attribute, unlike `tuiCell` which is required for directive to match.

Reply:
fixed